client/signup/steps/import-from/index.tsx

```diff
@@ -10,10 +10,14 @@ import './style.scss';
 interface Props {
 	stepName: string;
 	stepSectionName: string;
+	queryObject: {
+		temp: string;
```

Comment:
I kind of find the name `temp` a bit weird, as it's usually the short form of `temporary`. What does it stand for here? If possible, can we find a better name? `stage` perhaps?

Reply:
It is temporary stuff, the temporary way to show both screens we made. The next step is to include the business logic and I'll remove the temp query param support for sure.